### DISCUSSION TITLE
Autosave document

### DISCUSSION THREAD

**Discussion by new-ranger** [ONLYOFFICE/DocumentServer#1792] [Q&A] [answered]
Hi,

I am using Onlyoffice with node JS application. I wanted to understand how 'Autosave' works. Do we need to close document editor to save the changes? How it works?

**Comment by jiriks74** [comment, marked as answer]
@new-ranger I cannot tell you how is the autosave written but from my experience (about 3 years of using it) it saves a copy to the DocumentServer's cache and once everyone closes the file then it saves the file.

EDIT: In the settings of Nextcloud there's a option to force save the document so it should be possible (I haven't tried it though)

**Reply by ShockwaveNN** [reply]
Yep, I think you're right

About saving to cache and about [`forcesave`](https://api.onlyoffice.com/editors/config/editor/customization#forcesave)
